test/data/team/team.yaml

```diff
@@ -11,4 +11,5 @@ spec:
     isDeploy: false
 status:
   namespace:
-    staging: default
\ No newline at end of file
+    staging: default
+    active: active-Ns
```

Comment:
if you want to use dash-case, should it be `active-ns`?

Reply:
and not sure why it's needed

Reply:
it used for webhook unit test